Error during publishing utbot-maven to maven local

**Description**

publishToMavenLocal gradle task is failed due to occurring several errors:
```
> Task :utbot-maven:generatePluginDescriptor FAILED
Execution failed for task ':utbot-maven:generatePluginDescriptor'.
> Process 'command 'C:\Program Files\Amazon Corretto\corretto-17.0.6\bin\java.exe'' finished with non-zero exit value 1
```

**To Reproduce**

Run `utbot-maven/publishing/publishToMavenLocal` gradle task

**Expected behavior**

Plugin is supposed to be published.

**Actual behavior**

```
Caused by: org.apache.maven.project.DependencyResolutionException: 
Could not resolve dependencies for project org.utbot:utbot-maven:maven-plugin:2023.03-SNAPSHOT: 
Could not find artifact org.utbot:utbot-fuzzing:jar:2023.03-SNAPSHOT in MavenRepo_0 (https://repo.maven.apache.org/maven2/)
```

--- a/utbot-maven/build.gradle
+++ b/utbot-maven/build.gradle
@@ -146,6 +146,7 @@ generatePluginDescriptor.dependsOn([
     project(':utbot-framework'),
     project(':utbot-framework-api'),
     project(':utbot-fuzzers'),
+    project(':utbot-fuzzing'),
     project(':utbot-rd'),
     project(':utbot-summary')
 ]*.tasks.publishToMavenLocal)

